src/people.js

```diff
@@ -26,6 +26,23 @@
  * @returns {PeopleResponse}  - Processed information
 */
 
-function peopleInformation(people) { }
+function peopleInformation(people) {
+    let avrAge = 0;
```

Comment:
Looks to use a reduce any time you are taking data, and "reducing" it down to something else